lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -0,0 +1,16 @@
+module ShopifyAPI
+  class FulfillmentOrder < Base
+    def self.all(options = {})
+      order_id = options.dig(:params, :order_id)
+      raise ShopifyAPI::ValidationException, "'order_id' is required" if order_id.nil? || order_id == ''
+
+      order = ::ShopifyAPI::Order.new(id: order_id)
+      order.fulfillment_orders
+    end
+
+    def fulfillments(options = {})
+      fo_fulfillments = get(:fulfillments, options)
```

Comment:
Anywhere you find yourself abbreviating fulfillment orders to fo, it's best to use the full phrase for clarity